src/server/index.jsx

```diff
@@ -104,14 +104,23 @@ server
 
       const headHelmet = Helmet.renderStatic();
 
-      const styleTags = sheet.getStyleElement();
+      const stylesheets = [...sheet.getStyleElement()];
```

Comment:
Could you give some context to why this now needs to be an array?

Reply:
Have added a comment. https://github.com/BBC-News/simorgh/pull/921/commits/bc3d462ef1f938807b80a44775d7e198b9b978db